app/views/corporate_information_page/show.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :title, "#{@presenter.page_title} - #{content_item.default_organisation.title} - GOV.UK" %>
```

Comment:
Again, in the original code they're testing if a default organisation is present. If it's possible one won't be, we should be testing too. If it's guaranteed to be there, we don't have to test but we should remove guard code in the previous commit.